src/components/pages/NavBarLanding/NavBarLanding.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Link, NavLink } from 'react-router-dom';
+import { Menu, Button, Layout, Breadcrumb } from 'antd';
+import './NavBarLanding.css';
+
+const { Header } = Layout;
+
+export default function NavBarLanding() {
```

Comment:
Good job on building the layout of the landing page, will continue refactor and add more code for the header and footer.